src/App.js

```diff
@@ -133,6 +134,9 @@ export default class App extends Base {
     } catch (e) {
       logger.error(JSON.stringify(e), 'init');
     }
+    if (!process.env.MACADDRESS) {
+      process.env.MACADDRESS = CONSTANTS.DEFAULT_MAC;
+    }
     // Get values from Parameter.initialization - End
```

Comment:
not able to understand the logic of assigning default mac at the starting of app launch itself. It should ONLY be assigned when everything else fails

Reply:
this is not correct

Reply:
removed the code